src/classes/agent.js

```diff
@@ -0,0 +1,18 @@
+import User from './user';
+import Traveler from './traveler';
+// import Trip from './trip';
+
+let foundTraveler;
```

Comment:
It's a little unclear to me why empty variables are getting instantiated within a class.